test_tools/src/test_tools/doctest_unittest.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python
+
+import doctest
+import importlib
+import os
+import rospkg
+import unittest
+
+
+class _TestDocTests(unittest.TestCase):
+
+    def __init__(self, module_name, method_name="test_doctests"):
+        super(_TestDocTests, self).__init__(method_name)
```

Comment:
Add docstring:
* `module_name`: is this only, e.g., `robot_smach_states`?
* what does the `method_name` do?

Reply:
The method name is the name of the member function to run.